README.md

```diff
@@ -4,12 +4,13 @@ FS2 Scalaz: Interoperability between FS2 and Scalaz
 [![Build Status](https://travis-ci.org/functional-streams-for-scala/fs2-scalaz.svg?branch=master)](http://travis-ci.org/functional-streams-for-scala/fs2-scalaz)
 [![Gitter Chat](https://badges.gitter.im/functional-streams-for-scala/fs2.svg)](https://gitter.im/functional-streams-for-scala/fs2)
 
-This library provides an interoperability layer between FS2 and Scalaz. At this time, the API of this library is two imports:
+This library provides an interoperability layer between FS2 and Scalaz. At this time, the API of this library is three imports:
 
     import fs2.interop.scalaz._         // Provides conversions from FS2 to Scalaz (e.g., FS2 Monad to Scalaz Monad)
     import fs2.interop.scalaz.reverse._ // Provides conversions from Scalaz to FS2 (e.g., Scalaz Monad to FS2 Monad)
+    import fs2.interop.scalaz.async._   // Provides `Async` and `Async.Run` instances for Scalaz `Task`
```

Comment:
I think we could put the task instances in the `fs2.interop.scalaz` object instead but I don't have strong feelings. What do you think?